health/notifications/health_alarm_notify.conf

```diff
@@ -284,6 +284,7 @@ SEND_OPSGENIE="YES"
 
 # Api key
 OPSGENIE_API_KEY=""
+OPSGENIE_API_URL="https://api.opsgenie.com"
```

Comment:
Please include all the changes from #10562 into this PR.

Don't forget to update [opsgenie documentation](https://github.com/netdata/netdata/tree/master/health/notifications/opsgenie#send-notifications-to-opsgenie) as well.

Reply:
I've removed this OK to leave unset now.
Updated documentation to reflect new variable.